wgsl/index.bs

```diff
@@ -3818,20 +3817,24 @@ and that subexpression is [=in scope=] of the variable's declaration.
 Note that being statically accessed is independent of whether an execution of the shader
 will actually evaluate the subexpression, or even execute the enclosing statement.
 
-More precisely, the <dfn noexport>interface of a shader stage</dfn>
-is the set of module-scope variables
-[=statically accessed=] by [=functions in a shader stage|functions in the shader stage=],
-and which are in storage classes [=storage classes/in=], [=storage classes/out=], [=storage classes/uniform=], [=storage classes/storage=], or [=storage classes/handle=].
+More precisely, the <dfn noexport>interface of a shader stage</dfn> consists of:
+  - all parameters of the entry point (as [=storage classes/in=])
+  - the result value of the entry point
+  - all global variables that are [=statically accessed=] by [=functions in a shader stage|functions in the shader stage=],
+    and which are in storage classes [=storage classes/uniform=], [=storage classes/storage=], or [=storage classes/handle=].
 
 ### Pipeline Input and Output Interface ### {#pipeline-inputs-outputs}
 
+Let's define <dfn dfn>Entry point IO type</dfn> to include the following:
```

Comment:
The `Let's` here makes it sound like this is an example. Maybe something like `The <dfn dfn>Entry point IO type</dfn>s include the following:`